content/en/docs/use-cases/bootstrapping/index.md

```diff
@@ -168,6 +168,9 @@ navigator.credentials.create({
       residentKey: "required",
       userVerification: "preferred"
     },
+    "extensions": {
+      "credProps": true
+    }
```

Comment:
This seems like an arbitrary addition without some explanatory text. Will something get added to explain why we recommend using this extension, and how to use its return value?

Reply:
we currently don't explain any of them. That is probably something we need to add more globally at a later date.

Reply:
What motived adding this, though? Not even #226 has context. If this is a "shut up Matt and approve" then lmk.

Reply:
I noticed it was missing :)